api/intake/intakeModel.js

```diff
@@ -0,0 +1,56 @@
+const db = require('../../data/db-config');
+
+const update = (id, changes) => {
+  return db('clients').where({ id }).update(changes, '*');
+};
+
+const updateEducationHistoryByClientId = (id, changes) => {
+  return db('education_histories')
+    .where({ client_id: id })
+    .update(changes, '*');
+};
+
+const updateLocationsByHouseholdId = (id, changes) => {
+  return db('locations').where({ household_id: id }).update(changes, '*');
+};
+
+const updateHouseholdByName = (name, changes) => {
```

Comment:
Why do we need to update household by name here? If we have already created the household, can we just query the households table by id instead?